src/core/dynamic-user.h

```diff
@@ -1,16 +1,22 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 #pragma once
 
+#include <stdbool.h>
+#include <stdio.h>
+#include <sys/types.h>
+
+#include "fdset.h"
+#include "memory-util.h"
+
 typedef struct DynamicUser DynamicUser;
```

Comment:
Please just move the decl of DynamicUser struct above then

Reply:
Pushed fixes on top of https://github.com/systemd/systemd/pull/37224